common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -627,7 +625,7 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V> {
 
         @Override
         public void remove() {
-            if (prevIndex < 0) {
+            if (prevIndex == -1) {
```

Comment:
@Scottmitch so we are sure prevIndex can never be < -1 ?  Just want to make sure we are :)